internal/authflow/flow.go

```diff
@@ -23,12 +22,13 @@ var (
 	oauthClientSecret = "34ddeff2b558a23d38fba8a6de74f086ede1cc0b"
 )
 
-func AuthFlowWithConfig(cfg config.Config, hostname, notice string, additionalScopes []string) (string, error) {
+func AuthFlowWithConfig(cfg config.Config, io *iostreams.IOStreams, hostname, notice string, additionalScopes []string) (string, error) {
 	// TODO this probably shouldn't live in this package. It should probably be in a new package that
 	// depends on both iostreams and config.
-	stderr := colorable.NewColorableStderr()
+	stderr := io.ErrOut
+	cs := io.ColorScheme()
 
-	token, userLogin, err := authFlow(hostname, stderr, notice, additionalScopes)
+	token, userLogin, err := authFlow(hostname, stderr, cs, notice, additionalScopes)
```

Comment:
Does it make sense to just send in `io` to `authFlow` instead of `stderr` and `cs` separately?